internal/layerfile/layerfile_test.go

```diff
@@ -128,10 +127,10 @@ func TestToLayers_ValidateNameOfLayerDefinitions(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			_, err := tt.lf.ToLayers()
-			if err != nil {
-				assert.EqualError(t, tt.err, err.Error())
+			if tt.err == nil {
+				assert.NoError(t, err)
 			} else {
-				assert.NoError(t, tt.err)
+				assert.ErrorIs(t, err, tt.err)
```

Comment:
Also, it's fine to just use `ErrorIs`, I don't want this test to fail if we change error messages.

Reply:
ok, got it